src/tls13.c

```diff
@@ -870,13 +957,16 @@ static const byte finishedLabel[FINISHED_LABEL_SZ+1] = "finished";
  * ssl     The SSL/TLS object.
  * key     The key to use with the HMAC.
  * secret  The derived secret.
+ * forSide The side that we are deriving the secret for.
```

Comment:
I originally had it as side, but it gets confusing because the client could set this to be for server. Anyways, if you don't mind that confusion, I'll change it back to side.

Reply:
Within this function side makes sense.
Context when called not so much.